Sources/Apodini/Utilities/TypeInfo.swift

```diff
@@ -36,6 +36,7 @@ func isOptional(_ type: Any.Type) -> Bool {
 
 // MARK: - Supported Scalar Types
 private let supportedScalarTypes: [ObjectIdentifier] = [
+    ObjectIdentifier(Int.self),
```

Comment:
This may be related to #140.
To my knowledge `supportedScalarTypes` strongly resembles what the Protobuffer Exporters support in theirs of primitive types. In fact, In my opinion such exporter specific information should be moved into the exporter itself.
However that's currently not the case and thus might break the current gRPC exporter(?). If I'm not wrong (help @nityanandaz)

Reply:
@nityanandaz whats your opinion on this? If you manage to support `Int` (as planned with the mentioned issue #140), would it be reasonable to include `Int` in supportedScalarTypes?